src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/api-authorization/api-authorization.constants.ts

```diff
@@ -0,0 +1,81 @@
+export const ApplicationName = 'Company.WebApplication1';
+
+export const ReturnUrlType = 'returnUrl';
+
+export const QueryParameterNames = {
+  ReturnUrl: ReturnUrlType,
+  Message: 'message'
+};
+
+export const LogoutActions = {
+  LogoutCallback: 'logout-callback',
+  Logout: 'logout',
+  LoggedOut: 'logged-out'
+};
+
+export const LoginActions = {
+  Login: 'login',
+  LoginCallback: 'login-callback',
+  LoginFailed: 'login-failed',
+  Profile: 'profile',
+  Register: 'register'
+};
+
+let applicationPaths : ApplicationPathsType = {
+  DefaultLoginRedirectPath: '/',
+  ApiAuthorizationClientConfigurationUrl: `/_configuration/${ApplicationName}`,
+  Login: `authentication/${LoginActions.Login}`,
+  LoginFailed: `authentication/${LoginActions.LoginFailed}`,
+  LoginCallback: `authentication/${LoginActions.LoginCallback}`,
+  Register: `authentication/${LoginActions.Register}`,
+  Profile: `authentication/${LoginActions.Profile}`,
+  LogOut: `authentication/${LogoutActions.Logout}`,
+  LoggedOut: `authentication/${LogoutActions.LoggedOut}`,
+  LogOutCallback: `authentication/${LogoutActions.LogoutCallback}`,
+  LoginPathComponents: [],
+  LoginFailedPathComponents: [],
+  LoginCallbackPathComponents: [],
+  RegisterPathComponents: [],
+  ProfilePathComponents: [],
+  LogOutPathComponents: [],
+  LoggedOutPathComponents: [],
+  LogOutCallbackPathComponents: [],
+  IdentityRegisterPath: '/Identity/Account/Register',
+  IdentityManagePath: '/Identity/Account/Manage'
+}
+
+applicationPaths = {
+  ...applicationPaths,
+  LoginPathComponents: applicationPaths.Login.split('/'),
+  LoginFailedPathComponents: applicationPaths.LoginFailed.split('/'),
+  RegisterPathComponents: applicationPaths.Register.split('/'),
+  ProfilePathComponents: applicationPaths.Profile.split('/'),
+  LogOutPathComponents: applicationPaths.LogOut.split('/'),
+  LoggedOutPathComponents: applicationPaths.LoggedOut.split('/'),
+  LogOutCallbackPathComponents: applicationPaths.LogOutCallback.split('/')
+};
+
+interface ApplicationPathsType {
+  readonly DefaultLoginRedirectPath: string,
```

Comment:
TSLint: `properties should be separated by semicolon` (on all lines from 59 to 78)

Reply:
Done